Dockerfile

```diff
@@ -3,5 +3,6 @@ FROM registry.opensource.zalan.do/stups/openjdk:latest
 EXPOSE 8443
 
 COPY zmon-controller-app/target/zmon-controller-1.0.1-SNAPSHOT.jar /zmon-controller.jar
+COPY private-libs private-libs
 
-CMD java $JAVA_OPTS $(java-dynamic-memory-opts) $(appdynamics-agent) -jar /zmon-controller.jar
+CMD java $JAVA_OPTS $(java-dynamic-memory-opts) $(appdynamics-agent) -cp /zmon-controller.jar -Dopentracing.provider=lightstep -Dloader.path=private-libs/ org.springframework.boot.loader.PropertiesLauncher
```

Comment:
Please remove the specific tracer form the Docker image. This should be enabled using env vars

Reply:
done